pkg/network/encoding/format.go

```diff
@@ -90,6 +96,23 @@ func FormatConnection(
 	return c
 }
 
+// FormatFailedConnection converts a FailedConnStats into a model.FailedConnection
+func FormatFailedConnection(failedConn network.FailedConnStats, ipc ipCache) *model.FailedConnection {
```

Comment:
Is this PR to include the updated `FailedConnStats` structure discussed in the `#net-failed-connections` channel?: https://dd.slack.com/archives/C044P246PFB/p1665674162480669

Something like:
```json
{
    "pid": 110443,
    "laddr": {
      "ip": "127.0.0.1",
      "port": 57308,
      "containerId": "",
      "hostId": "0",
      "hostName": ""
    },
    "raddr": {
      "ip": "127.0.0.1",
      "port": 10000,
      "containerId": "",
      "hostId": "0",
      "hostName": ""
    },
    "family": "v4",
    "type": "tcp",
    "direction": "outgoing",
    "netNS": 4026531840,
    "failureReason": // one of the reasons we track
  }
```